spec/controllers/news_item_controller_spec.rb

```diff
@@ -2,4 +2,26 @@
 
 RSpec.describe NewsItemsController, type: :controller do
 
+  context 'when accessing a published news_item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.